.travis.yml

```diff
@@ -20,22 +19,23 @@ jdk:
 
 install: true
 
-script: ./test.sh
+script: 
+  - ./.buildscript/test.sh
 
 after_success:
-  - .buildscript/deploy_snapshot.sh
+  - ./.buildscript/deploy_snapshot.sh
 
 env:
   global:
     - secure: jt1Zcx55jOaZZIbTRdLfe89Uoof3WdHyNylETQzl13M/AdXBRrnmNPiC9nRJaa0thdAbRjtqTCcZ5bkP3eCEywIJ3kaLSUoPzbccH0BMUFFdXBPqwGgY2+XdDJumFPeu/Mg4LyGMN3pzwaOkIql+Wm12up/16P0AfAF8NuA1aSL8Vslmh93tU8Rm0W7jsm1KapaWhH1JF8smKu5UX325GSMpROpze7P0uVz0iqdtqhx4kRtNsMLdwFNaXzovWWIQ4gdoQqVr9tyo3I6o9jptPftcIlqtQoS3S0xqD9ekyh8WlTUkBj0zBn1RKbNI8v4n9qvZ6i4IwL3T446jcjwA2sUrMtdUS+YDVfqJT5PHa7ojaaXg/k0YZ9YyATVY6kjxofDYOAmkU97B+cN7T5Mv42huUjsRyEv8M5xoSm3ls2qgaiAmlXxL/DtFuzXdBTbEkgGu2N1GdgvlOhquzO9IzKQZ8JoJiUUQ9Tl5NxXSjZU3SoRMazw04zN40Q6C/gYJ1hV23VGq7bUQN2we5TSBejOEMgn6ND8oPyKSdqReA+ekwd2EjCYacqTgyvCkWg+dZKPUXbK9lUxuQploiBuOY7WJSv4ANnfeJoxbMx9lydB3ks/J4skuEjc9v20WO74xavsSm7eANJ28xayxqWg3cPJCPd/XVDCt49Hqo0VfZ28=
     - secure: Y3UuQXQ6+NQQa7RMp7hf88Biv5VfcPmdz/y1hPaNWg/Ux5xKV9E3CqTCbjJs+dZP8xKo/DLARnDQxUEiBTzmAXOvCGinI9iYgW2zRohhKWWKQ9a/O2cIevXgOJ5pIMTCAEac68ysQpHRw6MtSg/Ha3rGDr0INtghDkvmv4Kn55fz7lun7Pp0koCufyZl1/NE0BgOS556Px4ZbPH4PpdNuD86Xxg2NJam26dPLu8jFvC4JxPfdylQEsR/fBmQtGRk9GLQOqYKH2scA3sipWTBI6SAMHVJScte5FKcnClump8My7NjrZCMT97rk7iETN97/IxWe2yVdFEqQOptYWzUMUJT3AppqtlvMxm5kTzPt5kLY2Xl8AITvOW3BCkmaTa4x3dQtu+fNf4jyFONSOuv/x4O0oTLOC3C8E5BmsGJsp+rok2Nbv6yFF3etj9ENH54jO9cpo4xFfDeSuvqi9pP5CsBQHzVt9Vi/3KafK+a1JvauskICI3+CcyKXLDZho2VNFFdRE/mqRcFBXa2wr7tzs7ic1P9nVRisv/PxmHBQeNsnYyiJUUVIh3jFN6M0VZPnRkE20YLwdWs7jqlsL+voipGkqSn1TGYj4wCt7H/+f6Om2mdnmcdrRVZ2kMyW6E0kILHdSiygPbEaaXpKZckQwr+Vku+zMck/OjvvpQRqqc=
 
-sudo: false
```

Comment:
Unless something has changed, `sudo: false` is required to use Travis' container-based features, including caching.  Is that no longer the case?